src/com/nutrons/nu17/subsystems/GearPlacerServo.java

```diff
@@ -6,7 +6,7 @@
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
- * Similar to GearPlacer class but uses a servo instead.
+ * Similar to {@link GearPlacer} class but uses a servo instead.
```

Comment:
Perhaps this should extend the GearPlacer class or alternately you should have an interface that abstracts over both.